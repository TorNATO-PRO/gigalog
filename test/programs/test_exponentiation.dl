power(X, Y, Z) :- Z = X ^ Y.

power(2, 3, 8).
power(5, 2, 25).
power(3, 3, 27).
