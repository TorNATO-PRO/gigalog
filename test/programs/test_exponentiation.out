power(2,3,8)
power(3,3,27)
power(5,2,25)
